usrp_drivers/usrp_driver.cpp

```diff
@@ -328,16 +330,16 @@ void transmit(zmq::context_t &driver_c, USRP &usrp_d, const DriverOptions &drive
               ); //pulse timeit macro
             }
 
-            // Read AGC and Low Power signals after the pulse is sent, waiting for agc_signal_read_delay
-            // to make sure the transmitter time-delay circuits have time to update the LP/AGC signals.
-            // TODO: Does this belong here or could it go in an out-of-band thread? Same with GPS lock info and time diff
+            // Read AGC and Low Power signals, bitwise OR to catch any time the signals are active
+            // during this sequence for each USRP individually
             usrp_d.clear_command_time();
             auto read_time = sequence_start_time + (seqtime * 1e-6) + agc_signal_read_delay;
             usrp_d.set_command_time(read_time);
-            // Read all motherboards' gpio bank status into 32-bit pin_status_h and pin_status_l
-            pin_status_h = usrp_d.get_gpio_bank_high_state();  // TODO: This currently overwrites itself every pulse
-            pin_status_l = usrp_d.get_gpio_bank_low_state();  // TODO: either place it below and check it once, or use a logical OR with prev values
-            usrp_d.clear_command_time(); // TODO: that way we can get one value for each pulse sequence, and only report one value per record in data_write
+            agc_status_bank_h = agc_status_bank_h | usrp_d.get_agc_status_bank_h();
```

Comment:
Is there a reason you OR this with zero instead of straight assigning the return?

Reply:
ah, it's originally zero but then you OR it with itself to catch any time that the AGC status was active during the integration period.

Reply:
Okay cool I got it now.